app/views/books/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<ul id = "subjects">
+   <% Subject.find_each do |c| %>
+   <li><%= link_to c.name, action: "show", controller: "subjects", id: c.id %></li>
+   <% end %>
+</ul>
+
+<% if @books.blank? %>
+<p>There are not any books currently in the system.</p>
+<% else %>
+<p>These are the currently books in our system</p>
+
+<ul id = "books">
+   <% @books.each do |c| %>
+   <li>
+    <%= link_to c.title, {action: 'show', id: c.id} %>
+    <b> <%= link_to 'Edit', {action: 'edit', id: c.id}%> </b>
```

Comment:
Doesn't require a chance but you can do the same `book_path` thing here. The show action could be `book_path(c)` and the edit action can be `edit_book_path(c)`. It helps others when reading the code to make the paths a but more semantic!